android/build.gradle

```diff
@@ -67,6 +66,12 @@ task javadocs(type: Javadoc) {
     }
 }
 
+afterEvaluate {
+    javadocs.classpath += files(android.libraryVariants.collect { variant ->
+        variant.javaCompile.classpath
```

Comment:
This is producing this warning. We shouldn't include new usages of `variant.javaCompile`.

> WARNING: API 'variant.getJavaCompile()' is obsolete and has been replaced with 'variant.getJavaCompileProvider()'.
It will be removed at the end of 2019.
For more information, see https://d.android.com/r/tools/task-configuration-avoidance.

Reply:
Done. The warning should go away after #6066 is merged.